docs/jwst/outlier_detection/index.rst

```diff
@@ -2,6 +2,52 @@
 Outlier Detection
 =================
 
+Processing multiple datasets together allows for the identification of bad pixels
+or cosmic-rays that remain in each of the input images, many times at levels which
+were ambiguous for detection during ramp fitting.  The outlier detection step 
+implements the following algorithm to identify and flag any remaining cosmic-rays or
+other artifacts left over from previous calibrations:
+
+  - build a stack of input data
+  
+    - all inputs will need to have the same WCS in order
+    - if needed, each input will be resampled to a common output WCS
+  
+  - create a median image from the stack of input data
+
+    - this median operation will ignore any input pixels which have a weight 
+      which is too low (<70% max weight)
+  
+  - create "blotted" data from the median image to exactly match each original
+    input dataset
+  
+  - perform a statistical comparison (pixel-by-pixel) between the median,blotted
+    data with the original input data to look for pixels with values greater 
```

Comment:
"greater" implies we're only looking for and flagging positive outliers. Is that true or does it also check for negative outliers?